code/__DEFINES/layers.dm

```diff
@@ -153,15 +154,15 @@
 #define CURSE_LAYER 22.5
 #define FULLSCREEN_RENDER_TARGET "FULLSCREEN_PLANE"
 
-#define HUD_PLANE 23
-#define HUD_LAYER 23
+#define HUD_PLANE 24
```

Comment:
Runechat shouldn't be on hud layer.
If you are moving stuff up one layer be consistent.